report-pr-errors

```diff
@@ -314,6 +317,22 @@ class Logs(object):
                                    .format(self.all_logs[-1], err)]
         self.preview_error_log = '\n'.join(error_log_lines).strip()
 
+    def get_versions(self):
+        # The versions are defined differently for both job runner setups
+        # Jenkins (daily builds):
+        #   - aliBuild: ALIBUILD_SLUG
+        #   - alidist: ALIDIST_SLUG (if not set it defaults to alidist@master)
+        # Nomad (CI checks):
+        #   - aliBuild: INSTALL_ALIBUILD
+        #   - alidist: (defaults to alidist@master)
+        self.alibuild_version = os.getenv('ALIBUILD_SLUG') or os.getenv('INSTALL_ALIBUILD') or 'Not pinned (most likely alibuild@master)'
+        alidist_version = os.getenv('ALIDIST_SLUG')
+        if alidist_version:
```

Comment:
I am not sure I understand the logic here. Why do you do two different things depending on if it is alibuild_version or alidist version?

Reply:
This is an old change from when I was getting the commit hash, which was easy to convert to a github URL. The new version is not, so I'll remove the github links